guides/plugins/plugins/framework/data-handling/entities-via-attributes.md

```diff
@@ -0,0 +1,411 @@
+---
+nav:
+  title: Entities via attributes
+  position: 1
+
+---
+
+# Entities via attributes
+
+Since shopware v6.6.3.0 it is possible to register entities via php attributes. This is a new way to register entities in Shopware 6. This guide will show you how to do this.
```

Comment:
```suggestion
Since Shopware v6.6.3.0 it is possible to register entities via PHP attributes. This guide will show you how to do this.
```

Removed redundant text